lib/github/contents_test.go

```diff
@@ -0,0 +1,23 @@
+package github_test
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/philips-labs/tabia/lib/github"
+)
+
+func TestDownloadContents(t *testing.T) {
+	assert := assert.New(t)
+	gh := github.NewClientWithTokenAuth(os.Getenv("TABIA_GITHUB_TOKEN"))
+	contents, err := gh.DownloadContents(context.Background(), "philips-labs", "tabia", "README.md")
+	if assert.NoError(err) {
+		readme, _ := ioutil.ReadFile("../../README.md")
+		assert.NotEmpty(contents)
+		assert.Equal(string(readme), string(contents))
+	}
+}
```

Comment:
Do you want to test error handling, e.g. non-existent file, as well?

Reply:
Good point. Let me add that.